src/app/components/chat/window/output/item/item.vue

```diff
@@ -6,6 +6,7 @@
 		:class="{
 			'chat-msg-type-normal': message.type === ChatMessage.TypeNormal,
 			'chat-msg-type-system': message.type === ChatMessage.TypeSystem,
```

Comment:
Here too, remove type system, not used.
Make sure to also remove the `chat-msg-type-system` declaration in the CSS file.